scripts/get_databaker_data.py

```diff
@@ -239,8 +238,7 @@ def main():
         os.makedirs(args.data_root)
 
     if args.dataset_name == 'databaker_csmsc':
-        __prepare_databaker_csmsc(
-            args.data_root, args.train_size, args.output_dir)
+        __prepare_databaker_csmsc(args.data_root, args.train_size)
```

Comment:
Why did we remove the output_dir option? Can we add it back with a default for the --output_dir command line argument?

Reply:
> Why did we remove the output_dir option? Can we add it back with a default for the --output_dir command line argument?

@blisc The output_dir option was originally designed to let users specify the dir where the manifests locate. But I found that in our document and get_librispeech_data.py the manifests are just located at data_root. So I think we can simply generate the manifest in data_root